app.js

```diff
@@ -23,6 +24,8 @@ const appRouter = express.Router();
 const adminApp = express();
 const adminAppRouter = express.Router();
 
+app.use(bodyParser.json());
```

Comment:
Can you load the middleware by using router instance and place it in the same section [here](https://github.com/osu-mist/express-api-skeleton/blob/master/app.js#L50-L51)?

```js
appRouter.use(bodyParser.json());
```

In addition, should this go to the skeleton as well?

Reply:
It looks like express-openapi has a [consumesMiddleware](https://www.npmjs.com/package/express-openapi#argsconsumesmiddleware) field where this goes. So the initialize function should look like this:
```js
initialize({
...
  consumesMiddleware: {
    'application/json': bodyParser.json(),
  },
...
});
```
This requires the openapi document to specify
```yaml
consumes:
  - application/json
```
in the `POST` path though, otherwise an error will be raised. I think this is fine though since it requires us to provide the MIME type of the endpoint explicitly.

Reply:
I think this would be fine to go in the skeleton, even though it doesn't have a POST endpoint, since I'm assuming the default handler for the `application/json` type would be `bodyParser.json()`.